src/Templating/Helper/BlockHelper.php

```diff
@@ -284,7 +284,8 @@ public function render($block, array $options = [])
         }
 
         if ($this->stopwatch) {
-            $stats['cache']['created_at'] = $response->getDate();
+            // avoid \DateTime because of serialize/unserialize issue in PHP7.3 (https://bugs.php.net/bug.php?id=77302)
+            $stats['cache']['created_at'] = $response->getDate()->getTimestamp();
```

Comment:
Is this change required for versions different than 7.3 too? I don't know how is it used, just asking to be sure if this change should modify the behavior for versions not affected by the issue too.
By other side, I see in the signature of `Response::getDate()` that it allows `null` as return value. Is that scenario not possible here?

Reply:
I don't know but since the definition allows null I've fixed for null return values 👍

Reply:
Thank you.
I've also shared another concern:
>Is this change required for versions different than 7.3 too? I don't know how is it used, just asking to be sure if this change should modify the behavior for versions not affected by the issue too.

Reply:
Sorry. Wanted to check this first again. The change is not required except for PHP 7.3 - but it seems that the `created_at` var isn't used in the templates at all. So maybe the best solution would have been to remove this line.

Reply:
Let's keep this as is then :+1: